data.sql

```diff
@@ -35,4 +35,44 @@ VALUES ('Blossom', '1998-10-13', 3, true, 17);
 
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES ('Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+VALUES ('Ditto', '2022-05-14', 4, true, 22);
```

Comment:
- [ ] Ido not see the following data that is supposed to be added to owners' table:
 - Sam Smith 34 years old.
 - Jennifer Orwell 19 years old.
 - Bob 45 years old.
 - Melody Pond 77 years old. 
 - Dean Winchester 14 years old.
 - Jodie Whittaker 38 years old.

Reply:
Data update